swmcc/swm.cc
# ✨ #29 - Starting - Project Current with Index Page and Navigation Updates

#### Summary:
This PR marks the initiation of Project Current, a feature aimed at providing real-time insights into various personal activities and interests. An index page for the 'Current' section has been created, and links to this new page have been integrated into the navigation bar and footer.

#### Changes:

1. **Created Index Page for 'Current':** A new index page has been added under the 'Current' section, serving as a gateway to various subsections such as "Reading," "Watching," "Coding," etc.

2. **Navbar Updates:** The navigation bar has been updated to include a link to the 'Current' section, providing easy access from any part of the website.

3. **Footer Updates:** Similar to the navigation bar, a link to the 'Current' section has also been added to the website's footer for better accessibility.

#### Testing:

- Verify that the 'Current' link appears in both the navigation bar and the footer.
- Confirm that the link redirects to the newly created index page for the 'Current' section.

#### Additional Notes:

This PR sets the foundation for Project Current, and further enhancements and features will be added incrementally. This is part of an ongoing public experiment; you can track the progress at [staging.swm.cc](https://staging.swm.cc) and on the issue list.
